tools/syz-covermerger/syz_covermerger.go

```diff
@@ -27,11 +27,13 @@ var (
 	flagDateTo        = flag.String("date-to", "", "[optional] used to mark DB records")
 	flagSaveToSpanner = flag.String("save-to-spanner", "", "[optional] save aggregation to spanner")
 	flagProjectID     = flag.String("project-id", "syzkaller", "[optional] target spanner db project")
+	flagJobs          = flag.Int64("jobs", 1, "[optional] jobs count")
```

Comment:
If it works well in the parallel mode, would you every need to run it in a single-threaded one? Maybe just always use https://pkg.go.dev/runtime#NumCPU?

Reply:
Thanks, agree.